src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1392,6 +1476,11 @@ A function that intercepts (reads and modifies) the `Response` of a `Request` or
 
 This is an optional argument of `onRequest()`, and may provide the required `Response` returned by the middleware.
 
+<Since v="4.13.0" />
+
+**Type:** `(rewritePayload?: string | URL | Request) => Promise<Response>`
+
+It accepts an optional URL path parameter in the form of a string, `URL`, or `Request` to trigger a [rewrite](/en/guides/routing/#rewrites) of the current request.
```

Comment:
```suggestion
<p><Since v="4.13.0" /></p>

**Type:** `(rewritePayload?: string | URL | Request) => Promise<Response>`

It accepts an optional URL path parameter in the form of a string, `URL`, or `Request` to [rewrite](/en/guides/routing/#rewrites) the current request without retriggering a new rendering phase.
```